benchmarks/oss.py

```diff
@@ -118,6 +117,10 @@ def train(
 ):
     logging.basicConfig(level=logging.INFO if not args.debug else logging.DEBUG)
 
+    use_multi_tensor = args.multi_tensor_optim and hasattr(torch.optim, "_multi_tensor")
+    OPTIM = torch.optim._multi_tensor.RMSprop if use_multi_tensor else torch.optim.RMSprop  # type: ignore  # attr is  checked but mypy misses that
```

Comment:
Have we tested this with OSS? I couldn't find an example.

Reply:
no, sorry if I was not clear, the whole point of this PR is to test that :)

Reply:
I guess I was confused by the CL description which said adding tests for new optims but I guess you meant regression benchmarks. 
 If you want to test the new optims, you could do a golden weight check which is a different test than the regression benchmark you are adding. I am not sure if we need to add a regression benchmark (mainly because it adds to the time taken to run all checks on GH). Given I don't have context about this use case, I'll leave you to decide :)